<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOptionsTab</class>
 <widget class="QWidget" name="GameOptionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>302</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>302</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>302</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10,10,10,0">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="musicLabel">
            <property name="text">
             <string>Volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="musicPercentlabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="musicHorizontalSlider">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="languageLabel">
          <property name="text">
           <string>Language:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="languageComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>musicHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>musicPercentlabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>musicHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GameOptionsTab</receiver>
   <slot>slotChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GameOptionsTab</receiver>
   <slot>slotChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChanged()</slot>
  <slot>slotLanguageChanged()</slot>
 </slots>
</ui>
